pkl-core/src/main/java/org/pkl/core/service/ExecutorSpiImpl.java

```diff
@@ -116,13 +116,22 @@ public String evaluatePath(Path modulePath, ExecutorSpiOptions options) {
             .setTimeout(options.getTimeout())
             .setOutputFormat(options.getOutputFormat())
             .setModuleCacheDir(options.getModuleCacheDir());
-    if (options.getProjectDir() != null) {
-      var project = Project.loadFromPath(options.getProjectDir().resolve(PKL_PROJECT_FILENAME));
-      builder.setProjectDependencies(project.getDependencies());
-    }
 
-    try (var evaluator = builder.build()) {
-      return evaluator.evaluateOutputText(ModuleSource.path(modulePath));
+    try {
+      if (options.getProjectDir() != null) {
+        var project =
+            Project.loadFromPath(
+                options.getProjectDir().resolve(PKL_PROJECT_FILENAME),
+                SecurityManagers.defaultManager,
```

Comment:
While we're here; this should really use the same security manager as the normal evaluator

```suggestion
                securityManager,
```